deploy/vagrant/modules/buttonmen/templates/create_databases.erb

```diff
@@ -6,7 +6,8 @@ if [ "$?" = "0" ]; then
 else 
   echo "Creating <%= @buttonmen_db1_name %>"
   echo "CREATE DATABASE <%= @buttonmen_db1_name %>" | mysql -u root
-  echo "GRANT ALL ON <%= @buttonmen_db1_name %>.* TO <%= @buttonmen_db1_user %>@localhost IDENTIFIED BY '<%= @buttonmen_db1_pass %>'" | mysql -u root
+  echo "CREATE USER <%= @buttonmen_db1_user %>@localhost IDENTIFIED WITH mysql_native_password BY '<%= @buttonmen_db1_pass %>'" | mysql -u root
```

Comment:
Bug i found in replay testing: users are persistent even when we drop the database, so this command errors out after the first time you run create_buttonmen_databases.  Not sure that's *why* replay-testing isn't working, but at the least it's an unnecessary error message.